src/transformers/models/funnel/modeling_tf_funnel.py

```diff
@@ -1127,12 +1127,12 @@ def call(
             training=training,
         )
 
-    # Copied from transformers.models.distilbert.modeling_tf_distilbert.TFDistilBertModel.serving_output
```

Comment:
Removing copied from across models as the tensors in the tuples for hidden_states and activations are different sizes so can't be called with `tf.convert_to_tensor`

Reply:
A quick look in Funnel docstring, I could not find why they have different sizes.

https://github.com/huggingface/transformers/blob/2c5747edfe383eee073119de784fa148befe9f2d/src/transformers/models/funnel/modeling_funnel.py#L843-L845

Maybe the docstring is wrong, and we should update it (in another PR for sure) ..?

Reply:
It seems because it uses pooling, the sequence length is different after each block: https://github.com/huggingface/transformers/blob/2c5747edfe383eee073119de784fa148befe9f2d/docs/source/en/model_doc/funnel.mdx?plain=1#L38